### ISSUE TITLE
[Wildcards] RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by modestaspruckus** [express-validator/express-validator#1205]
## Describe the bug

When escaping payload fields with wildcard `*`, it throws error:

```js
RangeError: Maximum call stack size exceeded
    at Array.slice (<anonymous>)
    at /Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:44:14
    at Array.map (<anonymous>)
    at expandPath (/Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:42:14)
    at /Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:50:13
    at Array.forEach (<anonymous>)
    at expandPath (/Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:49:14)
    at /Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:50:13
    at Array.forEach (<anonymous>)
    at expandPath (/Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:49:14)
    at /Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:50:13
    at Array.forEach (<anonymous>)
    at expandPath (/Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:49:14)
    at /Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:50:13
    at Array.forEach (<anonymous>)
    at expandPath (/Users/modestaspruckus/projects/backend/node_modules/express-validator/src/select-fields.js:49:14)

```
## To Reproduce

1. Go to https://runkit.com/modestaspruckus/63c7f7aacca97c00089cd8b6
2. Click on `run`

## Expected behavior

No error
## Current behavior

Throws `RangeError: Maximum call stack size exceeded`
## Environment:

- Express-validator version: 6.14.0
- Express version: 4.18.2
- Node.js version: 16.17.1

**Comment by gustavohenke** [comment]
Thanks! I've reproduced it and will work on a fix.

**Comment by gustavohenke** [comment]
Fix published v6.14.3.
